src/Tizen.WebView/Tizen.WebView/ContextMenu.cs

```diff
@@ -0,0 +1,488 @@
+        /// <summary>
+        /// Show substitutions.
+        /// </summary>
+        ShowSubstitutions,
+        /// <summary>
+        /// Smart copy paste.
+        /// </summary>
+        SmartCopyPaste,
+        /// <summary>
+        /// Smart quotes.
+        /// </summary>
+        SmartQuotes,
+        /// <summary>
+        /// Smart dashes.
+        /// </summary>
+        SmartDashes,
+        /// <summary>
+        /// Smart links.
+        /// </summary>
+        SmartLinks,
+        /// <summary>
+        /// Text replacement.
+        /// </summary>
+        TextReplacement,
+        /// <summary>
+        /// Transformation menu.
+        /// </summary>
+        TransformationMenu,
+        /// <summary>
+        /// Make upper case.
+        /// </summary>
+        MakeUpperCase,
+        /// <summary>
+        /// Make lower case.
+        /// </summary>
+        MakeLowerCase,
+        /// <summary>
+        /// Capitalize.
+        /// </summary>
+        Capitalize,
+        /// <summary>
+        /// Change back.
+        /// </summary>
+        ChangeBack,
+        /// <summary>
+        /// Open media in new window.
+        /// </summary>
+        OpenMediaInNewWindow,
+        /// <summary>
+        /// Copy media link to clipboard.
+        /// </summary>
+        CopyMediaLinkToClipboard,
+        /// <summary>
+        /// Toggle media controls.
+        /// </summary>
+        ToggleMediaControls,
+        /// <summary>
+        /// Toggle media loop.
+        /// </summary>
+        ToggleMediaLoop,
+        /// <summary>
+        /// Enter video fullscreen.
+        /// </summary>
+        EnterVideoFullscreen,
+        /// <summary>
+        /// Media play pause.
+        /// </summary>
+        MediaPlayPause,
+        /// <summary>
+        /// Media mute.
+        /// </summary>
+        MediaMute,
+        /// <summary>
+        /// Dictation alternative.
+        /// </summary>
+        DictationAlternative,
+        /// <summary>
+        /// Select all.
+        /// </summary>
+        SelectAll,
+        /// <summary>
+        /// Select word.
+        /// </summary>
+        SelectWord,
+        /// <summary>
+        /// Text selection mode.
+        /// </summary>
+        TextSelectonMode,
+        /// <summary>
+        /// Clipboard.
+        /// </summary>
+        Clipboard,
+        /// <summary>
+        /// Drag.
+        /// </summary>
+        Drag,
+        /// <summary>
+        /// Translate.
+        /// </summary>
+        Translate,
+        /// <summary>
+        /// Copy link data.
+        /// </summary>
+        CopyLinkData,
+        /// <summary>
+        /// If app want to add customized item, use enum value after BaseApplicationTag.
+        /// </summary>
+        BaseApplicationTag = 10000
+    }
+
+    /// <summary>
+    /// This class provides the properties of Context Menu item.
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public class ContextMenuItem
```

Comment:
ContextMenuItem should be inherited by EventArgs.

Reply:
Done